test/base/test_batch.py

```diff
@@ -322,8 +322,10 @@ def test_utils_to_torch():
     assert batch_torch_float.a.dtype == torch.float32
     assert batch_torch_float.b.c.dtype == torch.float32
     assert batch_torch_float.b.d.dtype == torch.float32
-    array_list = [float('nan'), 1.0]
-    assert to_torch(array_list).dtype == torch.float64
+    data_list = [float('nan'), 1]
+    data_list_torch = to_torch(data_list)
+    assert data_list_torch[0].dtype == torch.float64
```

Comment:
Is this platform dependent? Or it surely returns data type with ``torch.float64``? The same for the following line ``torch.int64``.

Reply:
It is not platform dependent. Default native Python and numpy type are using 64 bits, while torch is doing the same for integers, but using float32 byt default for floating points, on any platforms.